codecov_cli/runners/dan_runner.py

```diff
@@ -10,10 +11,19 @@
 class DoAnythingNowConfigParams(dict):
```

Comment:
Are the ConfigParams classes between different runners have the same behavior/properties? if so can we have a Base class for them?

Reply:
No, they are independent.

The idea of the runners is similar to the plugins, in that every plugin is (and should be) completely independent and self-contained.

I don't think we should try to impose any interface on the runner's config. Each has its own needs. The details of those needs are irrelevant from the CLI's perspective.